app/routes/index.js

```diff
@@ -310,39 +265,86 @@ const storage = multer.diskStorage({
 			else
 				res.redirect('/admin');
 		});
-	app.post('/adduser', upload.single('profile_pic'), async function(req,res){
+	app.route('/adduser').post(adminAuth, async function(req,res){
 		try {
-			const user = new User(req.body);
-			console.log(user)
-			if(req.file){
-				user.profile_pic = req.file.tempFilePath;
-			}
-			await user.save()
-			res.json("User added successfully");
+			const form = new formidable.IncomingForm();
+
+    	form.parse(req, async function(err, fields, files){
+		console.log(fields)
+		console.log(files)
+		
+		const user = new User(fields);
+        var oldPath = files.profile_pic.filepath;
+		console.log(__dirname)
+        var newPath = path1.join(__dirname, '/../../upload')+ '/'+fields.username
```

Comment:
Please use `path.join` consistently to join path items. 
```suggestion
var newPath = path1.join(__dirname, '/../../upload', fields.username)
```